e2e/tests/LoggedIn/livePublication.e2e.spec.ts

```diff
@@ -102,28 +102,6 @@ test.describe('Live Publication', () => {
         await testFlagging(page, 'cl3fz14dr0001es6i5ji51rq4', 'testing the flagging functionality');
     });
 
-    test('Author profile', async ({ browser }) => {
```

Comment:
There was no need for this to be in the logged in tests so I have moved it to profile.e2e.spec.ts.